lib/runner.js

```diff
@@ -49,11 +51,17 @@ app.on('ready', function() {
    */
 
   parent.on('browser-initialize', function(options) {
+    var userPreload = resolve('userPreloadTemplate.js');
```

Comment:
Adding a fix to do something akin to...

```javascript
if('{{!userPreloadFilepath}}' != ''){
    require('{{!userPreloadFilepath}}')(__nightmare.ipc);
}
```

... removes the need for the placeholder or try/catch.  I didn't like this at first because it makes the generated file logic look... awkward?  but the more I think about it, the better it seems.